fixes/mss/20200609-msshome-brand

```diff
@@ -12,5 +12,11 @@ fi
 glib-compile-schemas /usr/share/glib-2.0/schemas/
 update-initramfs -u
 if [ $(arch) != x86_64 ]; then
-    echo "todo pi splash part" 
+    if ! grep -q plymouth; then
+        sed -i -e 's@$@ quiet splash plymouth.ignore-serial-consoles@' /boot/firmware/cmdline.txt
+    fi
+    cp /boot/firmware/initrd.img /boot/firmware/initrd.img.$(date +%m%d%H%M)
```

Comment:
Backup and copying over to /boot/firmware is being taken care of by `update-initramfs -u`. So the following 4 lines are redundant.